modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,227 @@
+stringData:
+  # The repository encryption key
+  RESTIC_PASSWORD: <secure-restic-password>
+----
++
+[NOTE]
+====
+By default, the operator looks for a secret named `dm-credential`. If you are using a different name, you need to specify the name through a Data Protection Application (DPA) CR using `dpa.spec.features.dataMover.credentialName`.
+====
+
+. Create a DPA similar to the following example. Note that the default plug-ins` include CSI.
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  features:
+    dataMover:
+      enable: true
+      credentialName: <secret-name> <1>
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <bucket-name>
+          prefix: <bucket-prefix>
+        provider: aws
+  configuration:
+    restic:
+      enable: <true or false>
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+----
+<1> Add the Restic secret name from the previous step. If this is not done, the default secret name `dm-credential` is used.
++
+The OADP operator installs two custom resource definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.
++
+.Example `VolumeSnapshotBackup` CR
++
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotBackup
+metadata:
+  name: <vsb-name>
+  namespace: <namespace-name> <1>
+spec:
+  volumeSnapshotContent:
+    name: <snapcontent-name>
+  protectedNamespace: <adp-namespace>
+  resticSecretRef:
+    name: <restic-secret-name>
+----
+<1> Specify the namespace where the volume snapshot resides.
++
+.Example `VolumeSnapshotRestore` CR
++
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotRestore
+metadata:
+  name: <vsr-name>
+  namespace: <namespace-name> <1>
+spec:
+  protectedNamespace: <protected-ns> <2>
+  resticSecretRef:
+    name: <restic-secret-name>
+  volumeSnapshotMoverBackupRef:
+    sourcePVCData:
+      name: <source-pvc-name>
+      size: <source-pvc-size>
+    resticrepository: <your-restic-repo>
+    volumeSnapshotClassName: <vsclass-name>
+----
+<1> Specify the namespace where the volume snapshot resides.
+<2> Specify the namespace where the Operator is installed. The default is `openshift-adp`.
+
+. To back up a snapshot:
+
+.. Create a backup CR:
+
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup-name>
+  namespace: <protected-ns> <1>
+spec:
+  includedNamespaces:
+  - <app-ns>
+  storageLocation: velero-sample-1
+----
+<1> Specify the namespace where the Operator is installed. The default is `openshift-adp`.
++
+.. Wait up to 10 minutes and check whether the `VolumeSnapshotBackup` CR status is `Completed`:
```

Comment:
This would help customers if this was labelled as a `.Verification` step as shown in the [Mod Docs procedure template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc).

Since you are working under a deadline, it might be better to take this up as a continuous improvment item (CIT) after your GA. You might need to refactor the ordered list of your procedure module. It might even involve making separate modules for the verification procedures.